.github/workflows/ci.yaml

```
@@ -191,7 +191,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
```

```suggestion
      matrix:
      # We only test on the lowest and highest supported PyPy versions
```

Or however you do comments in yaml